helix-term/src/ui/menu.rs

```diff
@@ -74,6 +74,7 @@ impl<T: Item> Menu<T> {
     // rendering)
     pub fn new(
         options: Vec<T>,
+        sort: bool,
```

Comment:
Using an enum here would be clearer, and we follow this pattern in most places in the codebase.

Reply:
I usually use an `bool` whenever its just about enableing or disabling a single option and an enum if there are two seperate orthogonal options.

However your comment drew my attention to the fact that sorting in `new` never makes sense (the fuzzy score of every item is the same and its an unstable sort so the order is prettymuch random) so I just reemoved that argument and never sort. This actually yields slightly nicer results when the autocompletions popup menu first shows up